week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -131,7 +133,10 @@ function getLocationName() {}
   Advanced challange: try to use arrow function when invoking an array method.
 */
 function journeyPlanner(locations, transportMode) {
-  // Implement the function body
+  const getLocationName = locations.filter((typesOfTransport) =>
+    typesOfTransport.includes(transportMode)
+  );
+  return getLocationName.map((locations) => locations[0]);
```

Comment:
Keeping code DRY

Hint: You have a function, you've completed, which you can reuse here on this line